functions/controllers/players.js

```diff
@@ -25,4 +25,24 @@ router.post("/", async (request, response) => {
   }
 });
 
+router.put("/", async (request, response) => {
+  try {
+    await playerService.updatePlayer(request.body);
+    response.status(200).send();
+  } catch (error) {
+    response.status(500).json(error.message).send();
+    return;
+  }
+});
+
+router.delete("/", async (request, response) => {
```

Comment:
I'd change this to just take playerId, probably in the URL and not need to pass any body, is it just needs the id to delete.
So send a DELETE to /api/players/123